quodlibet/docs/packaging.rst

```diff
@@ -117,6 +117,8 @@ plugin from loading.
 **libappindicator-gtk3** + **typelibs**:
     * For the Tray Icon plugin under Ubuntu Unity and KDE Plasma
 
+**pydub**:
```

Comment:
(It's a bit unfortunate that pydub is needed, since theoretically we already ship everything for transcoding with GStreamer, but we can always change it at some later point)